README.rst

```diff
@@ -28,4 +28,10 @@ Usage
 
 Run::
 
-    python manage.py bootstrap
+    python manage.py bootstrap <file>
```

Comment:
Does roadbuster need to be updated to work with this?
Also does it mean that you have to run bootstrap multiple times in order to get both all of the files?

Reply:
Yes and yes.

It's possible to let users specify multiple files, but in current configuration that will cause -e switch to apply to all of specified files (so you can't use builtin roles file and your own path/to/file.json file in one command). I can investigate if it's possible to do it differently.